app/src/components/SkeletonLoading.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+  <div class="skeleton w-full h-96"></div>
```

Comment:
i think its better to make this component generic and reusable.
for eg, we can have width, height and other attributes passed down as props